packages/types/src/type/UncleEntryItem.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BlockNumber } from '.';
```

Comment:
Please do `import BlockNumber from './BlockNumber'` to avoid circular refs. (index importing this, this importing index)